inc/Engine/Optimization/RUCSS/Database/Schemas/UsedCSS.php

```diff
@@ -164,5 +164,15 @@ class UsedCSS extends Schema {
 			'date_query' => true,
 			'sortable'   => true,
 		],
+
+		// NOT_PROCEED_BEFORE column.
+		[
+			'name'       => 'not_proceed_before',
+			'type'       => 'timestamp',
+			'default'    => null,
```

Comment:
This comment might not be relevant: Just in case, maybe it would be safer to default to 0 so that we're sure not to have error when evaluating` not_proceed_before < current_time`? OR maybe PHP is OK with `null < current_time`?

Reply:
PHP est ok avec `null < (int)`